### ISSUE TITLE
Handling DateTimeField field change option

### ISSUE THREAD

**Issue by S2606** [TrangPham/django-admin-confirm#22]
So on setting up confirm screen on model change, while changing a DateTime field via the admin, this is the issue I am currently facing.

![Screenshot 2021-03-24 at 5 32 17 PM](https://user-images.githubusercontent.com/18497513/112307379-e34fc080-8cc6-11eb-8cbe-4bbcece53583.png)

`models.py`

```
class Polls(models.Model):
    poll_name = models.CharField(max_length=200, null=True, blank=True)
    poll_no = models.PositiveIntegerField(null=True, blank=True)
    start_date = models.DateTimeField(null=True, blank=True)
```

`admin.py`

```
@register(Polls)
class PollsAdmin(AdminConfirmMixin, admin.ModelAdmin):
    list_display = ("poll_name", "poll_no")
    search_fields = ("poll_name", )
    confirm_change = True
    confirmation_fields = ['poll_name', 'poll_no', 'start_date']
```

**Comment by S2606** [comment]
Also just found out that datetime fields are not supported I guess by reading [testing readme](https://github.com/TrangPham/django-admin-confirm/blob/main/admin_confirm/tests/README.md)

**Comment by TrangPham** [comment]
Hi @S2606 

What version of django_admin_confirm, python, and django are you using?
Could you post the stacktrace for your error? 

I tried to replicate the issue and was not able to do so. 

Thanks!

**Comment by S2606** [comment]
I was trying this on django_admin_confirm version number 0.2.2, python 3.7, django 2.2.5. But just found out that a new version is in pre-release(0.2.3a0). And this issue is not getting reproduced in that version.

**Comment by TrangPham** [comment]
Yay! Okay awesome. I’m glad. I was starting to add tests for it yesterday.

Let me know if you have any issues with v0.2.3a0 or if you’re using it
smoothly. If all goes well, I’ll release the pre-release as stable.

**Comment by TrangPham** [comment]
> Affected Release: [v0.2.2](https://github.com/TrangPham/django-admin-confirm/releases/tag/v0.2.2)
Fixed in: [v0.2.3a0](https://pypi.org/project/django-admin-confirm/0.2.3a0/)
